builder/source/git.go

```diff
@@ -125,19 +115,9 @@ func clone(uri, path, ref string) error {
 func reset(path, ref string) error {
 	fetchArgs := []string{
 		"fetch",
-		"--depth",
-		"1",
 		"origin",
 	}
 
-	// We have to add the tag keyword if the ref is a tag, otherwise
-	// git won't actually fetch the tag
-	if len(ref) != 40 {
-		fetchArgs = append(fetchArgs, "tag")
-	}
-
-	fetchArgs = append(fetchArgs, ref)
-
```

Comment:
I'm 90% sure that this part is no longer needed due to not using shallow clones so I removed it. If it turns out to be needed a better alternative would be to add `--tags` to the `git fetch` command instead.

Reply:
I agree with your 90% confidence :wink: